pom.xml

```diff
@@ -89,15 +90,15 @@
     <io.cucumber.version>5.6.0</io.cucumber.version>
     <de.monochromata.cucumber.version>4.0.42</de.monochromata.cucumber.version>
     <jackson2.version>2.9.9</jackson2.version>
-    <junit.version>5.5.1</junit.version>
+    <junit.version>5.8.2</junit.version>
```

Comment:
Additional changes nee to be made to support moving to JUnit 5 and have Cucumber work.

https://medium.com/codex/bdd-testing-with-cucumber-junit-5-fb5a1c4354f9